jumanji/environments/packing/bin_pack/generator.py

```diff
@@ -1311,3 +1314,375 @@ def _unpack_items(self, state: State) -> State:
         )
         state.items_placed = jnp.zeros(placed_item_arr_dims, bool)
         return state
+
+
+class ExtendedTrainingGenerator(ExtendedRandomGenerator):
+    def __init__(
+        self,
+        max_num_items: int,
+        max_num_ems: int,
+        mean_item_value: float,
+        std_item_value: float,
+        min_target_volume: int,
+        max_target_volume: int,
+        split_eps: float = 0.3,
+        prob_split_one_item: float = 0.7,
+        split_num_same_items: int = 5,
+        container_dims: Tuple[int, int, int] = TWENTY_FOOT_DIMS,
+    ):
+        """
+        Args:
+            is_rotation_allowed: whether the generator has to generate instances where item
```

Comment:
update docstring

Reply:
It would also be helpful to explain that the target volume is expressed in terms of nb of containers.